src/cascadia/TerminalApp/AppCommandlineArgs.h

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+#include "ActionAndArgs.h"
+
+#pragma once
+#include "Commandline.h"
+
+// fwdecl unittest classes
+namespace TerminalAppLocalTests
+{
+    class CommandlineTest;
+};
+
+namespace TerminalApp

```

Comment:
Why is this one fwddecl'd and then immediately normal decl'd? I get the above one.

Reply:
because you can't just `class Namespace::ClassName {} ` in C++, you have to `namespace NS { class Classname {}}` and that adds another level of indenting to the whole file and I don't love that so I typically avoid it and do this instead

Reply:
Oh. I get it. Yes OK. Resolving.